src/main/java/net/runelite/rsb/methods/Bank.java

```diff
@@ -606,7 +596,7 @@ public boolean open() {
 					didAction = bankBooth.doAction("Bank") || bankBooth.doAction("Use-Quickly");
 				} else if (banker != null) {
 					didAction = banker.doAction("Bank", "Banker");
-				} else if (bankChest != null) {
+				} else {
```

Comment:
Isn't this necessary? What if none of them are found? In that case its going to print a stack trace where are before it would just return false.

Reply:
In the lines above the bankBooth is set to null if the bankChest is not null. Therefore this is always true.